package.json

```
@@ -19,7 +20,7 @@
     "@mui/icons-material": "^5.11.9",
```

```suggestion
    "@mui/icons-material": "5.11.9",
```
https://the-guild.dev/blog/how-should-you-pin-dependencies-and-why